packages/nimbus/src/components/button/button.mdx

```diff
@@ -510,23 +456,17 @@ Button groups are UI elements that organize multiple buttons together, often
 visually as a single unit. They serve to present users with a set of related
 choices or actions in a compact and organized way.
 
-<Tip tipColor="#46A557" tipBackgroundColor="#F4FAF5" icon={<Icons.Lightbulb />}>
-  <div>
-    <strong>Do</strong>
-    <ul style={{ listStyle: "disc", marginLeft: "20px" }}>
-      <li>
-        This is the correct hierarchy of button groupings. From right to left,
-        Primary, Outlined, and Link button.
-      </li>
-    </ul>
-  </div>
-</Tip>
+> [!TIP]\
+> **Do**
+>
+> This is the correct hierarchy of button groupings. From right to left,
+> Primary, Outlined, and Link button.
```

Comment:
Should be in a list

Reply:
Yes, I noticed if it is just an item, it does not display the list when it is just one item in the list except we add it this way which is clearly wrong

<img width="773" alt="image" src="https://github.com/user-attachments/assets/325f27c9-1dd0-4df2-aafe-6b806f981e22" />

Vs

<img width="775" alt="image" src="https://github.com/user-attachments/assets/76bc40e8-957c-476b-b72f-84fe7c29c105" />

Vs

<img width="780" alt="image" src="https://github.com/user-attachments/assets/d14a33aa-0a81-4269-ac5b-e50d92823d3e" />

Reply:
Hahahahaha great, leave it as is!